internal/plugin/README.md

```diff
@@ -155,7 +155,7 @@ change in the near future_
 ## EventNotifier
 
 `EventNotifier` is used as a target object of events for handlers and listeners that notifies the plugin manager
-in an abstract way without needing to pass down the full connection manager as a parameter. 
+in an abstract way without needing to pass down the full connection manager as a parameter.
```

Comment:
In general I think we agreed as a group that the EventNotifier construct was also confusing. It may be that our lack of a clear group understanding of it led to an unclear description as well. This is not part of the new interface, though an updated iteration of it may reappear when/if we add a plugin to support managing connections again.